integration/presto/README.md

```diff
@@ -82,6 +82,29 @@ Please follow the below steps to query carbondata in presto
   For example, if you have a schema named 'default' stored in hdfs://namenode:9000/test/carbondata/,
   Then set carbondata-store=hdfs://namenode:9000/test/carbondata
   
+#### Connecting to carbondata store on s3
+ * In case you want to query carbonstore on S3 using S3A api put following additional properties inside $PRESTO_HOME$/etc/catalog/carbondata.properties 
+   ```
+    fs.s3a.access.key={value}
+    fs.s3a.secret.key={value}
+    Optional: fs.s3a.endpoint={value}
```

Comment:
better to describe the properties as below to make it convenient for user to copy&paste.
```
# Required properties
A=B
# Optional properties
C=D
```

Reply:
done